modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc

```diff
@@ -9,7 +9,23 @@ Clusters using a restricted network must import the default must-gather image to
 
 .Procedure
 
-* Import the default must-gather image from your installation payload:
+. If you have not added your mirror registry's trusted CA to your cluster's image configuration object as part of the Samples Operator configuration, do so:
```

Comment:
_do so_ seems a bit terse to me. _do the following_, _perform the following steps_?